### PR TITLE
Merge name and email fields

### PR DESCRIPTION
**Pull request by icanberk**

### CODE REVIEW

File: app/views/assessments/_gradesheet.html.erb [hunk 1]
```diff
@@ -25,8 +25,7 @@
 <thead>
     <tr class="sortable">
         <th class="enumerator">#</th>
-        <th class="id">Email</th>
-        <th>Name</th>
+        <th class="id">Email & Name</th>
```
Comment (jez) [inline app/views/assessments/_gradesheet.html.erb]: Should this be `&amp;`?

File: app/views/assessments/_gradesheet.html.erb [hunk 2]
```diff
@@ -50,14 +49,17 @@
...
+        <a tabindex="-1" class="email"><%= s.course_user_datum.email %></a>
+
         <% if s.filename %>
           <%=raw(view_file(s, go_img(-1), go_img(-1), go_img(-1))) %>
         <% end %>
         <div class="popover">
         </div>
-    </td>
-  <td><%= s.course_user_datum.last_name %>, <%= s.course_user_datum.first_name %></td>
+
```
Comment (jez) [inline app/views/assessments/_gradesheet.html.erb]: Trailing whitespace?

File: app/views/assessments/_gradesheet.html.erb (hunk 1 above)
Comment (icanberk) [inline app/views/assessments/_gradesheet.html.erb]: Does it really matter?

File: app/views/assessments/_gradesheet.html.erb (hunk 2 above)
Comment (icanberk) [inline app/views/assessments/_gradesheet.html.erb]: I'll cleanup now

File: app/views/assessments/_gradesheet.html.erb (hunk 1 above)
Comment (jez) [inline app/views/assessments/_gradesheet.html.erb]: Maybe it doesn't but the [Internet seems to think it does](http://stackoverflow.com/questions/3493405/do-i-really-need-to-encode-as-amp). Up to you.

**Comment by icanberk** [comment]
fixed

File: app/views/assessments/_gradesheet.html.erb [hunk 3]
```diff
@@ -50,14 +49,16 @@
 <tr class="<%= aud_special_grade_type?(s.aud) && @cud.instructor? ? "special_submission" : "" %>">
   <td class="enumerator"></td>
   <td class="id" data-submission-id="<%= s.id.to_s %>">
-        <a tabindex="-1" class="email" href="#"><%= s.course_user_datum.email %></a>
+        <%= s.course_user_datum.last_name %>, <%= s.course_user_datum.first_name %>
```
Comment (jez) [inline app/views/assessments/_gradesheet.html.erb]: It'd maybe be nice to make this ordering (Last, First vs First Last, etc.) configurable somehow, but it's probably fine when used with the search feature for now.

**Pull request merged**